### PR TITLE
Weekly will now require Java 17

### PR DESCRIPTION
**Pull request by psiinon**
Pending https://github.com/zaproxy/zaproxy/pull/8690 and the weekly release..

### CODE REVIEW

**Review by thc202 — APPROVED** [review]

**Comment by psiinon** [comment]
Not pending the latest weekly, as the last weekly required Java 17 😁

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

File: site/content/faq/what-versions-of-java-are-supported.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ weight: 2
 ---
 
 ZAP should be able to run with all/newer Java versions, but might require a minimum for certain ZAP versions:
+- ZAP 2.16.0 and later will require a minium of Java 17
```
Comment (ricekot) [inline site/content/faq/what-versions-of-java-are-supported.md:9]: typo: minium -> minimum

Comment (psiinon) [reply site/content/faq/what-versions-of-java-are-supported.md:9]: Good spot: https://github.com/zaproxy/zaproxy-website/pull/2850

**Pull request merged**